packages/components/src/date-time/date/styles.ts

```diff
@@ -90,8 +90,27 @@ export const DayButton = styled( Button, {
 		${ ( props ) =>
 			props.isSelected &&
 			`
-			background: ${ COLORS.theme.accent };
-			color: ${ COLORS.white };
+				background: ${ COLORS.theme.accent };
+
+				&,
+				&:hover:not(:disabled, [aria-disabled=true]) {
+					color: ${ COLORS.white };
```

Comment:
Not sure how relevant this still is, but since we're here, using a semantic color here may be better for eventual theming.

```suggestion
					color: ${ COLORS.theme.foregroundInverted };
```

Reply:
Sounds good!